types/accounting/accounting-tests.ts

```diff
@@ -3,6 +3,9 @@
 // Default usage:
 accounting.formatMoney(12345678); // $12,345,678.00
 
+// Stringified usage:
+accounting.formatMoney('$4394958309392.9401'); // $4,394,958,309,392.94
```

Comment:
If you follow the link I give above, you see the string parsing code under `formatMoney` via `unformat`.  We use this successfully in our (private) codebase.